manifest/manifest.go

```diff
@@ -127,6 +127,16 @@ func (m Manifest) Validate() []error {
 				errors = append(errors, fmt.Errorf("%s links to service: %s which does not expose any ports", entry.Name, l))
 			}
 		}
+
+		// TODO test cpu_shares
+
+		// test mem_limit
+		// FIXME: Do we accept text labels or not?
+		mem_as_str := fmt.Sprintf("%#v", entry.Memory)
+		mem, err := strconv.Atoi(mem_as_str)
```

Comment:
I believe this conversion is to check the value is between with 4 and 10,000.  Since Memory is really an int64, it can be used directly `(entry.Memory < 4 && entry.Memory != 0)`